File: checker_test/wa/submission.f90
program submission
    implicit none
    integer :: n
    integer :: i
    integer, allocatable :: a(:)
    integer :: max_val, max_idx
    
    read(*,*) n

    allocate(a(n))

    read(*,*) a

    max_val = a(1)
    max_idx = 1

    do i = 2, n
        if (a(i) > max_val) then
            max_val = a(i)
            max_idx = i
        end if
    end do

    print "(I0)", -1

    deallocate(a)
end program submission

File: testlib_checker_test/wa/submission.pas
Program Submission;
var
    n, i, max, maxIndex: integer;
    arr: array[1..1000] of integer;
begin
    readln(n);
    max := -1;
    for i := 1 to n do
    begin
        read(arr[i]);
        if arr[i] > max then
        begin
            max := arr[i];
            maxIndex := i;
        end;
    end;
    writeln(-1);
end.